tests/test_api.py

```diff
@@ -140,35 +140,37 @@ def test_folder_nested_files():
 
 
 def test_zipfile_unsupported_file():
-    with tempfile.TemporaryDirectory() as tmp_dir:
-        f = open(tmp_dir + "/unsupported_file.txt", "a")
+    with tempfile.TemporaryDirectory() as tmp:
+        f = open(tmp + "/unsupported_file.txt", "a")
         f.write("No geographical data")
         f.close()
-        with tempfile.NamedTemporaryFile(suffix=".zip") as tmp:
-            create_zip(tmp_dir, tmp)
-            result = geoextent.fromDirectory(tmp.name, bbox=True, tbox=True)
-            assert "bbox" not in result
-            assert "tbox" not in result
+        zip_path = tmp + "/zipfile.zip"
+        create_zip(tmp, zip_path)
+        result = geoextent.fromDirectory(zip_path, bbox=True, tbox=True)
+        assert "bbox" not in result
+        assert "tbox" not in result
 
 
 def test_zipfile_one_file():
     folder_name = "tests/testdata/folders/folder_one_file"
-    with tempfile.NamedTemporaryFile(suffix=".zip") as tmp:
-        create_zip(folder_name, tmp)
-        result = geoextent.fromDirectory(tmp.name, bbox=True, tbox=True)
+    with tempfile.TemporaryDirectory() as tmp:
+        zip_path = tmp + "/zipfile.zip"
+        create_zip(folder_name, zip_path)
+        result = geoextent.fromDirectory(zip_path, bbox=True, tbox=True)
         assert result["bbox"] == pytest.approx([7.601680, 51.948814, 7.647256, 51.974624], abs=tolerance)
         assert result["crs"] == "4326"
         assert result["tbox"] == ['2018-11-14', '2018-11-14']
 
 
 def test_zipfile_nested_folders():
     folder_name = "tests/testdata/folders/nested_folder"
-    with tempfile.NamedTemporaryFile(suffix=".zip") as tmp:
-        create_zip(folder_name, tmp)
-        result = geoextent.fromDirectory(tmp.name, bbox=True, tbox=True)
-        assert result["bbox"] == pytest.approx([7.601680, 34.7, 142.0, 51.974624], abs=tolerance)
-        assert result["crs"] == "4326"
-        assert result["tbox"] == ['2017-04-08', '2020-02-06']
+    with tempfile.TemporaryDirectory() as tmp:
+        zip_path = tmp+"/zipfile.zip"
```

Comment:
This might break on some systems with different path formatting, please only use `os.path.join()`.